File: JavaApp/src/main/java/com/charlezz/javaapp/feature/local/Person.java
package com.charlezz.javaapp.feature.local;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "person")
public class Person {

    @PrimaryKey(autoGenerate = true)
    private long id;
    private String name;
    private String birth;

    public Person(long id, String name, String birth) {
        this.id = id;
        this.name = name;
        this.birth = birth;
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getBirth() {
        return birth;
    }

    public void setBirth(String birth) {
        this.birth = birth;
    }
}

File: KotlinApp/src/main/java/com/charlezz/finalarchitecture/feature/local/Person.kt
package com.charlezz.finalarchitecture.feature.local

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "person")
data class Person
constructor(
        @PrimaryKey(autoGenerate = true)
        val id: Long,
        val name: String,
        val birth: String
)